_posts/2022-02-09-debezium-1-9-alpha2-released.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  Debezium 1.9.0.Alpha2 Released
+date:   2022-02-09
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: anmohant
+---
+
+It's my pleasure to announce the second release of the Debezium 1.9 series, *1.9.0.Alpha2*!
+
+This release includes support for Oracle 21c,
+improvements around Redis for Debezium Server, configuring `KAFKA_QUERY_TIMEOUT`,
+and a number of bug fixes around DDL parsers, build infrastructure, etc.
+
+Overall, the community fixed https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20%3D%201.9.0.Alpha2%20ORDER%20BY%20issuetype%20DESC[51 issues] for this release. Let’s take a closer look at some of the highlights.
+
++++<!-- more -->+++
+
+== Support for Oracle 21c
+
+The Debezium Oracle connector now supports the Oracle 21.3.0 with tested LogMiner and Xstreams compatibility.
+
+== Configuring `KAFKA_QUERY_TIMEOUT`
+
+When making API calls using Kafka Admin Client the default timeout was set to 3 seconds.
+But having this smaller timeout was insufficient for large TLS or SSL encryption, where the requests got timeout before completion.
+Now we have `KAFKA_QUERY_TIMEOUT` as a configurable field, it can be used to provide the wait time for fetching information from the Kafka Admin Client,
+while the default timeout remains 3 seconds.
+
+Thanks to the great work done by community member, https://github.com/snigdhasjg[Snigdhajyoti Ghosh].
+
+== Improvements in Redis for Debezium Server
+
+We have three new fields in the Redis support for Debezium Server
+
+* `redis.retry.initial.delay.ms`
+* `redis.retry.max.delay.ms`
+* `batch.size`
+
+The `maxmemory` configuration in Redis allows to provide a memory limit.
+If `maxmemory` is not configured, Redis keeps allocating memory which gradually eats up all free memory.
+This results in OOM errors and resolves as soon as entries are removed from the stream.
+`redis.retry.initial.delay.ms` and `redis.retry.max.delay.ms` is used in setting initial and max retry delays when encountering Redis connection or OOM issues.
+
+Using pipeline transactions helps in speeding up Redis queries.
+`batch.size` is used to provide the number of change records to insert in a single batch write.
```

Comment:
```suggestion
In order to leverage pipeline-based transactions, the `batch.size` configuration option can be specified which will allow Redis to write batches of change records rather than each record one by one.
```